css-three-column-layout/styles.css

```diff
@@ -0,0 +1,64 @@
+body {
+  background-color: #1996da;
+  font-family: Lato, sans-serif;
+  color: white;
+  text-align: center;
+}
+
+h1 {
+  font-weight: 200;
+  font-size: 2rem;
+  margin-top: 2em;
+  margin-bottom: 1em;
+}
+
+h3 {
+  font-weight: 500;
+  font-size: 1.4rem;
+  margin-block-end: 0;
+}
+
+img {
+  width: 50%;
+  background-color: white;
+}
+
+p {
+  color: #dbdcdd;
+  font-weight: 400;
+  margin-block-start: 0.75rem;
```

Comment:
I would stick to `margin-top`, this is a property primarily used in the User Agent Stylesheet primarily (default styling)